web/ui/react-app/src/Panel.tsx

```diff
@@ -157,29 +153,29 @@ class Panel extends Component<PanelProps, PanelState> {
       if (json.status !== 'success') {
         throw new Error(json.error || 'invalid response JSON');
       }
-
+      
       let totalTimeSeries = 0;
-      if (json.data !== undefined) {
+      if (json.data) {
         if (json.data.resultType === "scalar") {
           totalTimeSeries = 1;
-        } else if(json.data.result !== null) {
+        } else if(json.data.result && json.data.result.length > 0) {
           totalTimeSeries = json.data.result.length;
```

Comment:
Zero value shouldn't be displayed?

Reply:
No they shouldn't and so shouldn't ever get here if there are 0 results, but it's just in case something odd happens, it then displays a 0, which would be correct.